frontend/tests/unit/components/ShareTweet.spec.ts

```diff
@@ -75,9 +75,9 @@ describe('ShareTweet.vue', () => {
     const ctaEl = wrapper.find('[data-test="cta"]');
     const defaultText = `I just used @beamerbridge to seamlessly and securely transfer #${transfer.sourceAmount.token.symbol} from ${transfer.sourceChain.name} to ${transfer.targetChain.name} in ${transfer.transferTimeSeconds} seconds! 🔥
 
-Unlock lightning-fast and secure bridging with Beamer today 💪💫  Now also live on Polygon zkEVM!
-https://app.beamerbridge.com/
-`;
+Unlock lightning-fast and secure bridging with Beamer today 💪💫 and don't miss out on Beamer's #layerupfest! #beam2L2
```

Comment:
layerupfest???

Reply:
thats the default text. Zebra is showed only when the transfer was actually subsidized.